www/jsonrpc/server.go

```diff
@@ -0,0 +1,110 @@
+package jsonrpc
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"github.com/pacviewer/jrpc-gateway/jrpc"
+	"net/http"
+	"time"
+
+	"github.com/pactus-project/pactus/util/logger"
+	pactus "github.com/pactus-project/pactus/www/grpc/gen/go"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+)
+
+type Server struct {
+	ctx        context.Context
+	cancel     context.CancelFunc
+	config     *Config
+	server     *http.Server
+	grpcClient *grpc.ClientConn
+	handlers   map[string]func(ctx context.Context, params json.RawMessage) (interface{}, error)
```

Comment:
I think `handlers` should be remove from here and moves to plugin package.

Reply:
https://github.com/pactus-project/pactus/pull/1183/commits/512d8dabf827c33e4f445b27ae9fd39f87beb40a